runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/actions/excel.ts

```diff
@@ -6,6 +6,8 @@ const excelActionsTranslations = {
     'Process.Details.Modeler.Actions.Excel.Open.Path.Title': 'Path / Workbook',
     'Process.Details.Modeler.Actions.Excel.Open.Worksheet.Title': 'Worksheet',
     'Process.Details.Modeler.Actions.Excel.GetCell.Label': 'Get cell',
+    'Process.Details.Modeler.Actions.Excel.GetCells.Label': 'Get cells',
+    'Process.Details.Modeler.Actions.Excel.CellCoordinates': 'An object or a list (e.g. { "A1": null, "C2": "" } or ["A1", "C2"])',
```

Comment:
I think that if we want to get, for example, 100 cells next to each other (vertical or horizontal/both), we should be able to indicate their range without listing each one separately

Reply:
sure
done